examples/language_modeling-tf.ipynb

```diff
@@ -0,0 +1,1391 @@
+       "      <th>5</th>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>Points , NRR = Net run rate . \\n</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>Citra Award for Best Leading Actress , for Cinta Pertama ( 1973 ) \\n</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "show_random_elements(datasets[\"train\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "CKerdF353l-o"
+   },
+   "source": [
+    "As we can see, some of the texts are a full paragraph of a Wikipedia article while others are just titles or empty lines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JEA1ju653l-p"
+   },
+   "source": [
+    "## Causal Language modeling"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "v5GTGKZS3l-q"
+   },
+   "source": [
+    "For causal language modeling (CLM) we are going to take all the texts in our dataset and concatenate them after they are tokenized. Then we will split them in examples of a certain sequence length. This way the model will receive chunks of contiguous text that may look like:\n",
+    "```\n",
+    "part of text 1\n",
+    "```\n",
+    "or \n",
+    "```\n",
+    "end of text 1 [BOS_TOKEN] beginning of text 2\n",
+    "```\n",
+    "depending on whether they span over several of the original texts in the dataset or not. The labels will be the same as the inputs, shifted to the left.\n",
+    "\n",
+    "We will use the [`distilgpt2`](https://huggingface.co/distilgpt2) model for this example. You can pick any of the checkpoints listed [here](https://huggingface.co/models?filter=causal-lm) instead:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "id": "-WGBCO343l-q"
+   },
+   "outputs": [],
+   "source": [
+    "model_checkpoint = \"distilgpt2\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5io6fY_d3l-u"
+   },
+   "source": [
+    "To tokenize all our texts with the same vocabulary that was used when training the model, we have to download a pretrained tokenizer. This is all done by the `AutoTokenizer` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "id": "iAYlS40Z3l-v"
+   },
+   "outputs": [],
+   "source": [
+    "from transformers import AutoTokenizer\n",
+    "\n",
+    "tokenizer = AutoTokenizer.from_pretrained(model_checkpoint, use_fast=True)"
```

Comment:
`use_fast` is set to `True` by default, did you put it to be explicit to the user?